setup.cfg

```diff
@@ -16,4 +16,4 @@ norecursedirs = .env data maps
 show_capture = True
 minversion = 3.5
 console_output_style = count
-addopts = --pylint --cov-branch --cov=supervisor --cov-report term --cov-report xml:cov.xml --instafail --isort
+addopts = --pylint --cov-branch --cov=supervisor --cov-report term --cov-report xml:cov.xml --instafail
```

Comment:
**question**: was isort giving you different results based on your dev and installed code?

Reply:
It was becoming complicated to tell isort to ignore certain files (like example and secrets), so I just nuked it from the test coverage. It's been such a picky child before, with the isort test conflicting with how vscode was sorting my imports, that I wanted to save those brain cells for other stuff :)